packages/scaffolder-cli/knip-report.md

```diff
@@ -0,0 +1,10 @@
+# Knip report
+
+## Unused dependencies (3)
+
+| Name                               | Location     | Severity |
+| :--------------------------------- | :----------- | :------- |
+| @backstage/plugin-scaffolder-react | package.json | error    |
```

Comment:
These are probably relevant to fix, might just need to move some of them to `devDepenendencies` if they're only used in tests for example.

Reply:
I could not get knip to recognize the `@backstage/` imports for some reason but these are definitely used.